exotica/doc/overview.rst

```diff
@@ -77,7 +77,7 @@ The system model provides an interface to answer kinematic queries. A query can
  - Name of the tip frame (Frame A)
  - Offset of the tip frame
  - Name of the base frame (Frame B)
- - Offset the of base frame
+ - Offset of the base frame
```

Comment:
While we are at it, let's make it semantically correct: `offset from` instead of `offset of`